# Fix for Arrow Acero library name in LAPIS-SILO v0.7.10.2
# This backports a fix from commit fefe4cc4 (later SILO versions)
# The library name changed from Acero:: to ArrowAcero:: in newer conan packages
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2671eb0b..766c6de8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,7 +110,7 @@ target_link_libraries(
         silolib
         PUBLIC
         Arrow::arrow_static
-        Acero::arrow_acero_static
+        ArrowAcero::arrow_acero_static
         ${Boost_LIBRARIES}
         ${duckdb_LIBRARIES}
         nlohmann_json::nlohmann_json
